example/README.md

```diff
@@ -1,17 +1,118 @@
 # HPA Example
 
-Deploy the example resources in [./resources.yaml] to a Kubernetes cluster:
+## Disclaimer
+This example is validated using the latest SignalFX HPA Adapter on kubernetes v1.20. <br/>
+The adapter implements the autoscaling api `v2beta1`, this version is not supported in kubernetes v1.22 and above. <br/>
+We'll also be using Splunk OpenTelemetry distribution to emit app metrics to our backend
```

Comment:
Does this refer to the Splunk OpenTelemetry distribution of the Collector (helm chart)?